mmv1/products/monitoring/UptimeCheckConfig.yaml

```diff
@@ -72,6 +72,7 @@ custom_code: !ruby/object:Provider::Terraform::CustomCode
   constants: templates/terraform/constants/monitoring_uptime_check_config.go.erb
   custom_import: templates/terraform/custom_import/self_link_as_name.erb
   post_create: templates/terraform/post_create/set_computed_name.erb
+  custom_delete: templates/terraform/custom_delete/monitoring_uptime_check_config.go.erb
```

Comment:
Adding a custom delete also seems to remove the sweeper

https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9429-old..auto-pr-9429

Can you maintain the sweeper going forward with a custom sweeper? (likely just copy and paste old code)

Reply:
Tried adding it - will see if I did it right

Reply:
@ScottSuarez  any idea where the copies of the sweeper file should live? https://googlecloudplatform.github.io/magic-modules/get-started/how-magic-modules-works/ did not seem to point me to the right place

Reply:
Try adding the file to `mmv1/third_party/terraform/services/monitoring`

Reply:
That worked with a little bit of tweaking